ci/travis/format.sh

```diff
@@ -65,7 +65,7 @@ tool_version_check "mypy" "$MYPY_VERSION" "$MYPY_VERSION_REQUIRED"
 
 if which clang-format >/dev/null; then
   CLANG_FORMAT_VERSION=$(clang-format --version | awk '{print $3}')
-  tool_version_check "clang-format" "$CLANG_FORMAT_VERSION" "7.0.0"
```

Comment:
We also need to update https://docs.ray.io/en/latest/getting-involved.html since it mentions 7.0.0 there

Reply:
will this break the dev's setup at the moment? (i.e. running scripts/format.sh will fail) maybe we should make an announcement and suggest how to upgrade clang for Mac and Linux developers.

Reply:
Running `scripts/format.sh` will give a warning about installed clang-format version does not match the expected version, assuming clang-format-7 is installed. I will make an announcement on Slack. I believe it should be easier to install clang-format-12 than clang-format-7, for people using homebrew or on Ubuntu 20.04.